### ISSUE TITLE
2.11.	Outstanding:MeD: Glossary issues (2.4.3, 1.3.1, 1.4.3) 

### ISSUE THREAD

**Issue by falcochip** [MeD-DMC/Snap-framework-MeD#28]
Issue: 

> 1. Learners can become very frustrated at having to re-navigate the pop-out from the top every time they select a letter. with JAWS and IE11. 
> 2. Another issue is that the sighted users can see right away that certain letters have no definitions. This is because those letters are grey, but they are still active links. Screen reader users do not have this option and have to select every letter, re-navigate to the definition section to learn if there is a term or a warning that there are no terms for that letter.
> 3. Another issue is that since these are active links (the light grey letters) the text fails colour contrast with a ration of 1.4 to 1. The ratio needs to be at least 4.5 to 1 for the text in the links.
> 4. Another concern is that there is an accordion with the list of abbreviations and their expanded names, and then the alphabet buttons is just a repetition of what is in the accordion. This is not a true use of a glossary which would normally contain terms and their definitions.

Note: Note this is not an issue with Chrome since the focus will remain on the letter after selecting it, but the user will still have to navigate through the rest of the alphabet to get to the terms and definitions

Solutions: 

> 1. One solution is to keep the focus on the letter that was selected, so that the user only has to continue from there (through the rest of the alphabet) to get to the definition(s).
> 2. Another solution is to have the definition immediately following the letter so there is almost no navigation. (sort of like a tabbed accordion)
> 3. Make all active links black text, currently the entire alphabet would be black since they are all active links.
> 4. Disable the link for all “empty” letters, but keep the letter displayed in grey. Add an instruction before the alphabet to indicate that only letters with links have terms and definitions.

**Comment by farheenhalani** [comment]
“Back to top” button was added which allows us to return to the letter we has selected. 
The fix of making the disabled letters hidden is only a partial fix. 
When using JAWS, the lack of contrast is ok since the greyed out buttons are inactive button, but the learner needs to have instructions at the beginning of the Glossary to let them know that only letters with content will be displayed. However, this provides an unequal experience since sighted users can use the mouse to select the greyed out buttons. This means that the contrast on disabled letters does not meet any level of accessibility, so these should not be active buttons.
Still fixes to be made.
